lib/public/views/QcFlagTypes/Create/QcFlagTypeCreationPage.js

```diff
@@ -83,12 +75,12 @@ const qcFlagTypeCreationComponent = (qcFlagTypeCreationModel) => {
 
         h(PanelComponent, [
             h(LabelPanelHeaderComponent, { for: 'color' }, 'Color'),
-            h('label.f5.mb2', 'Please fill in a hexadecimal color after the #'),
+            h('label.f5.mb2', 'Select color'),
             colorInputComponent(color, (color) => qcFlagTypeCreationModel.patchFormData({ color })),
         ]),
     ];
 
-    return h('.w-25', panel);
+    return h('.w-l', panel);
```

Comment:
Another important thing is to add flex gap (using flex-column I suppose)

Reply:
added